firefox-ios/firefox-ios-tests/Tests/XCUITests/IntegrationTests.swift

```diff
@@ -63,9 +63,10 @@ class IntegrationTests: BaseTestCase {
         userState.fxaPassword = ProcessInfo.processInfo.environment["FXA_PASSWORD"]!
         navigator.performAction(Action.FxATypeEmail)
         navigator.performAction(Action.FxATapOnContinueButton)
+        mozWaitForElementToExist(app.staticTexts["Enter your password"], timeout: TIMEOUT_LONG)
         navigator.performAction(Action.FxATypePasswordExistingAccount)
         navigator.performAction(Action.FxATapOnSignInButton)
-        sleep(3)
+        mozWaitForElementToNotExist(app.staticTexts["Enter your password"], timeout: TIMEOUT_LONG)
```

Comment:
Do you need this call a second time? I think my test passed locally without it!

Reply:
The `sleep(3)` was here to ensure there's enough time to get back to the new tab screen. I just replaced it with `mozWait` so that we may wait shorter or longer depending on how the Mozilla Account server doing at the moment.